core/src/main/scala/org/apache/spark/shuffle/hash/BlockStoreShuffleFetcher.scala

```diff
@@ -17,23 +17,22 @@
 
 package org.apache.spark.shuffle.hash
 
-import scala.collection.mutable.ArrayBuffer
-import scala.collection.mutable.HashMap
+import java.io.InputStream
+
+import scala.collection.mutable.{ArrayBuffer, HashMap}
 import scala.util.{Failure, Success, Try}
 
 import org.apache.spark._
-import org.apache.spark.serializer.Serializer
 import org.apache.spark.shuffle.FetchFailedException
 import org.apache.spark.storage.{BlockId, BlockManagerId, ShuffleBlockFetcherIterator, ShuffleBlockId}
 import org.apache.spark.util.CompletionIterator
 
-private[hash] object BlockStoreShuffleFetcher extends Logging {
-  def fetch[T](
+private[shuffle] object BlockStoreShuffleFetcher extends Logging {
```

Comment:
Does that mean this file should live in the shuffle package rather than the hash package (if yes, can we do both the increased visibility and the move in a separate PR after this one)?